src/Arcus.Templates.EventHubs/Arcus.Templates.EventHubs.csproj

```diff
@@ -57,7 +57,7 @@
     <PackageReference Include="Arcus.Security.Core" Version="1.8.1" />
     <PackageReference Include="Arcus.Security.Providers.AzureKeyVault" Version="1.8.1" />
     <PackageReference Include="Guard.NET" Version="3.0.0" />
-    <PackageReference Include="Microsoft.Extensions.Hosting" Version="6.0.1" />
+    <PackageReference Include="Microsoft.Extensions.Hosting" Version="7.0.0" />
```

Comment:
Are these .NET 7 packages ?
If so, is this compatible with our current library ?

Reply:
No they are not.
Waiting until .NET 7 is out of preview. Will this PR on on-hold.